src/ONYXKEYS.ts

```diff
@@ -1,5 +1,6 @@
 import type {ValueOf} from 'type-fest';
 import type CONST from './CONST';
+import type {OnboardingPurposeType} from './CONST';
```

Comment:
I think it would be better if we move `OnboardingPurposeType` to `types/onyx`, don't you think?